samples/zigbee/light_bulb/prj.conf

```diff
@@ -43,3 +39,7 @@ CONFIG_NET_IPV6_NBR_CACHE=n
 CONFIG_NET_IPV6_RA_RDNSS=n
 CONFIG_NET_IP_ADDR_CHECK=n
 CONFIG_NET_UDP=n
+
+# BLE stack is required for MPSL compatible flash driver.
```

Comment:
This is still not fixed? 😕 This special flash driver shall be enabled by MPSL if it's required by it, please push MPSL guys to sort it out.

Reply:
@rlubos The MPSL compatible flash driver PR is here: https://github.com/nrfconnect/sdk-nrf/pull/3067

Reply:
Enabling BT fills all the flash size for some OT samples. We are hoping this gets merged on time: https://github.com/nrfconnect/sdk-nrf/pull/3067

Reply:
@eugmes Thanks for the info, great!

@tomchy Perhaps we should postpone this PR until #3067 gets merged, I see it's targeting 1.4.

Reply:
They work on it right now ;)
https://github.com/nrfconnect/sdk-nrf/pull/3067

Reply:
Well - if we wait for this PR, than the second commit has to be removed.

Reply:
The workaround removed from the PR. Now the #3067 has to be merged prior this PR.